apps/etterlatte-saksbehandling-ui/client/src/components/meldtInnEndring/MeldtInnEndring.tsx

```diff
@@ -0,0 +1,158 @@
+import { Link, useParams } from 'react-router-dom'
+import { useSidetittel } from '~shared/hooks/useSidetittel'
+import { StatusBar } from '~shared/statusbar/Statusbar'
+import { useApiCall } from '~shared/hooks/useApiCall'
+import { hentOppgave } from '~shared/api/oppgaver'
+import { useContext, useEffect, useState } from 'react'
+import { Alert, BodyShort, Button, Heading, HStack, Label, VStack } from '@navikt/ds-react'
+import { mapResult } from '~shared/api/apiUtils'
+import Spinner from '~shared/Spinner'
+import { ApiErrorAlert } from '~ErrorBoundary'
+import { hentJournalpost } from '~shared/api/dokument'
+import { ExternalLinkIcon } from '@navikt/aksel-icons'
+import { HjemmelLenke } from '~components/behandling/felles/HjemmelLenke'
+import { ConfigContext } from '~clientConfig'
+import { NyttBrevModal } from '~components/person/brev/NyttBrevModal'
+import { erOppgaveRedigerbar } from '~shared/types/oppgave'
+import { useInnloggetSaksbehandler } from '~components/behandling/useInnloggetSaksbehandler'
+import { MeldtInnEndringSkjema } from '~components/meldtInnEndring/MeldtInnEndringSkjema'
+
+export enum MeldtInnEndringHandlingValgt {
+  AVSLUTT_OPPGAVE,
+  OPPRETT_REVURDERING,
+  INGEN,
+}
+
+export const MeldtInnEndring = () => {
+  useSidetittel('Meldt inn endring')
+
+  const { oppgaveId } = useParams()
+  const configContext = useContext(ConfigContext)
+  const innloggetSaksbehandler = useInnloggetSaksbehandler()
+
+  if (!oppgaveId) {
+    return <Alert variant="error">Oppgave ID ligger ikke med i URL</Alert>
+  }
+
+  const [meldtInnEndringHandlingValgt, setMeldtInnEndringHandlingValgt] = useState<MeldtInnEndringHandlingValgt>(
+    MeldtInnEndringHandlingValgt.INGEN
+  )
+
+  const [hentOppgaveResult, hentOppgaveFetch] = useApiCall(hentOppgave)
+  const [hentJournalpostResult, hentJournalpostFetch] = useApiCall(hentJournalpost)
+
+  useEffect(() => {
+    hentOppgaveFetch(oppgaveId!, (oppgave) => hentJournalpostFetch(oppgave.referanse!))
+  }, [oppgaveId])
+
+  return (
+    <>
+      {mapResult(hentOppgaveResult, {
+        pending: <Spinner label="Henter oppgave..." />,
+        error: (error) => <ApiErrorAlert>{error.detail || 'Kunne ikke hente oppgave'}</ApiErrorAlert>,
```

Comment:
```suggestion
        error: (error) => <ApiErrorAlert>Kunne ikke hente oppgave, på grunn av feil: {error.detail}</ApiErrorAlert>,
```
Siden du garantert får detail på error, hjelper det ikke å ha det i en `||` som fallback. liker bedre å ha en kort tekst som beskriver hva som gikk galt, og så spy ut feilmeldingen fra backend